docs/DeveloperGuide.md

```diff
@@ -128,10 +203,10 @@ The following sequence diagram illustrates the sequence of interactions involved
 
 ## User Stories
```

Comment:
The user stories should be expanded upon, as there are clearly a variety of uses for this application.